packages/feedback-service/src/resolver.ts

```diff
@@ -0,0 +1,94 @@
+const Feedback_UPDATE = "Feedback_UPDATE";
+
+export const FeedbackResolver = {
+  Subscription: {
+    // subscriptions
+    updatedFeedbackData: {
+      subscribe: () => pubsub.asyncIterator(Feedback_UPDATE)
+    },
+  },
+  Query: {
+    // queries
+    listFeedback(root: any, args: any, ctx: any) {
+      return Feedback.find()
+        .populate("module")
+        .sort({ "timestamp": -1 })
+        .exec()
+        .then(response => response)
+        .catch(err => err);
+    },
+    getFeedback(root: any, args: any, ctx: any) {
+      return Feedback.findById(args._id)
+        .populate("module")
+        .then(response => response).catch(err => err);
+    }
+  },
+  Mutation: {
+    // mutations
+    addFeedback(root: any, args: any, ctx: any) {
+      const data = new Feedback(args.input);
+      return data.save().then(response => {
+        Feedback.findById(response._id)
+          .populate("module")
+          .then(fb => {
+            if (fb) {
+              const feedbackfor = fb.portalFeedback ? "One Platfrom" : Object(fb.module).name + " module";
+              const feedbackType = fb.feedbackType === "Bug" ? "submitting a bug report" : "your feedback";
+              let body, imsg, summary;
+              if (fb.iid) {
+                imsg = "You can track updates for your feedback via Jira at: Need to paste Jira URL" + fb.iid;
+              } else {
+                imsg = "";
+              }
+              if (fb.description) {
+                summary = "\nSummary: " + fb.description + "\n";
+              } else {
+                summary = "";
+              }
+              if (fb.feedbackType === "Feedback") {
+                // tslint:disable-next-line:max-line-length
+                body = `Hi ${fb.timestamp.createdBy.name},\n\nThanks for sharing your feedback on ${feedbackfor} with us.\n\nPlease find the details of the feedback we recieved as below:\nExperience: ${fb.experience} ${summary}\nThe feedback is currently active and we will be looking into it shortly.\n\nIn the meanwhile, you can also read the feedback shared by others here: https://one.redhat.com/feedback \n\n${imsg}\n\nRegards,\nOne Portal Team \n\nP.S.: This is an automated email. Please do not reply.`;
+              } else if (fb.feedbackType === "Bug") {
+                // tslint:disable-next-line:max-line-length
+                body = `Hi ${fb.timestamp.createdBy.name},\n\nThanks for submitting your bug report on ${feedbackfor} with us.\n\nPlease find the details of the feedback we recieved as below:\nTitle: ${fb.title} ${summary}\nThe bug is currently under review and we will be looking into it shortly.\n\nIn the meanwhile, you can also read the feedback shared by others here: https://one.redhat.com/feedback \n\n${imsg}\n\nRegards,\nOne Portal Team \n\nP.S.: This is an automated email. Please do not reply.`;
+              }
+
+              const environmentPaths: any = getEnvironmentPaths();
```

Comment:
environmentPaths should be defined at the top (commonly) so that u can use it for other mutations, queries or subscriptions

Reply:
Actually this is being handled by the `.env` config.
So this entire environmentPaths can be removed.